docs/configuration/index.md

```diff
@@ -1135,6 +1135,7 @@ The following configs only apply if the Overlord is running in remote mode. For
 |`druid.indexer.runner.taskAssignmentTimeout`|How long to wait after a task has been assigned to a Middle Manager before throwing an error.|`PT5M`|
 |`druid.indexer.runner.minWorkerVersion`|The minimum Middle Manager version to send tasks to. The version number is a string. This affects the expected behavior during certain operations like comparison against `druid.worker.version`. Specifically, the version comparison follows dictionary order. Use ISO8601 date format for the version to accommodate date comparisons. |"0"|
 | `druid.indexer.runner.parallelIndexTaskSlotRatio`| The ratio of task slots available for parallel indexing supervisor tasks per worker. The specified value must be in the range `[0, 1]`. |1|
+|`druid.indexer.runner.taskSlotRatiosPerWorker`|Map from task type to the ratio of task slots that the given task type can occupy on a single worker. Each value must be in the range [0, 1]. A value of 0 for a task type means that it can take up no task slot on a worker (i.e. tasks of this type would never run). A value of 1 means that the task type may take up all task slots of a worker if available. For `index_parallel` tasks, the `parallelIndexTaskSlotRatio` overrides the value specified in this map.|Empty map i.e. no limit on task slots for any task type|
```

Comment:
This might not solve the user problem if its per worker. It has to be global. 
Some implementations do container packing. The approach will fail there.

Reply:
Please join dev discussion at : https://apachedruidworkspace.slack.com/archives/C030CMF6B70/p1727836829640959